openapi_python_client/parser/properties/property.py

```diff
@@ -1,8 +1,27 @@
-from typing import ClassVar, Optional, Set
+from typing import ClassVar, NewType, Optional, Set
 
 import attr
 
-from ... import utils
+from ... import Config
+from ...utils import fix_keywords, fix_reserved_words, sanitize, snake_case
+
+_PythonIdentifier = NewType("_PythonIdentifier", str)
+
+
+def to_valid_python_identifier(*, value: str, prefix: str) -> _PythonIdentifier:
```

Comment:
Why was this moved out of `utils` with all the other case conversion logic? Seems clearer to be there, IMO.

Reply:
To keep the private constructor as private as possible, since the type is used directly here.